plugins/wp-songs-library/includes/class-wp-songs-library-cli.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Class Wp_Songs_Library_Cli
+ */
+class Wp_Songs_Library_Cli extends WP_CLI_Command {
+	public function __construct() {
+		WP_CLI::add_command( 'wsl', $this );
+	}
+
+	public function generate_sample_posts( $args, $args_assc ) {
+		$count = absint( $args[0] );
+		if ( empty( $args[0] ) || ! $count ) {
+			\WP_CLI::error( 'Provide the number of posts to generate.' );
+		}
+
+		$options = array(
+			'return'     => true,  // Return 'STDOUT'; use 'all' for full object.
+			'parse'      => 'json', // Parse captured STDOUT to JSON array.
+			'launch'     => true, // Reuse the current process.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://54.213.144.90:9000/sonar/coding_rules#rule_key=php%3AS139)